Dockerfile

```diff
@@ -1,14 +1,64 @@
-FROM mhart/alpine-node:10 as build-stage
+FROM node:18-alpine AS base
+
+# Install dependencies only when needed
+FROM base AS deps
+# Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
+RUN apk add --no-cache libc6-compat
+WORKDIR /app
+
+# Install dependencies based on the preferred package manager
+COPY package.json yarn.lock* package-lock.json* pnpm-lock.yaml* ./
+RUN \
+  if [ -f yarn.lock ]; then yarn --frozen-lockfile; \
+  elif [ -f package-lock.json ]; then npm ci; \
+  elif [ -f pnpm-lock.yaml ]; then yarn global add pnpm && pnpm i --frozen-lockfile; \
+  else echo "Lockfile not found." && exit 1; \
+  fi
+
+
+# Rebuild the source code only when needed
+FROM base AS builder
 WORKDIR /app
+COPY --from=deps /app/node_modules ./node_modules
 COPY . .
-RUN yarn install --pure-lockfile
-RUN yarn build
-RUN yarn --production
 
-FROM mhart/alpine-node:base-10
+# Next.js collects completely anonymous telemetry data about general usage.
+# Learn more here: https://nextjs.org/telemetry
+# Uncomment the following line in case you want to disable telemetry during the build.
+# ENV NEXT_TELEMETRY_DISABLED 1
+
+RUN NEXTJS_OUTPUT=standalone yarn build
```

Comment:
Are you sure this is correct? [Docs](https://nextjs.org/docs/pages/api-reference/next-config-js/output#automatically-copying-traced-files) mention only `standalone` option.

Reply:
Yes the docs neither seem to mention the available options nor the default, but without standalone the severs.js seems to be missing